diff --git a/doc/docbook_man.debian.xsl b/doc/docbook_man.debian.xsl
index f6027aa..6c498bc 100644
--- a/doc/docbook_man.debian.xsl
+++ b/doc/docbook_man.debian.xsl
@@ -5,5 +5,5 @@
 
   <xsl:strip-space elements="*"/>
 
-  <xsl:include href="file:///usr/share/xml/docbook/stylesheet/docbook-xsl/manpages/docbook.xsl"/>
+  <xsl:include href="file:///usr/share/xml/docbook/stylesheet/nwalsh/current/manpages/docbook.xsl"/>
 </xsl:stylesheet>

